### PR TITLE
Enhancement: Package name correction

### LINKED ISSUE #1133: [Installer Enhancement]: Correction of package name and publisher
## Description

- Upgrade package name to `PDF Arranger`.
- Add package publisher.

## Reproduction

- Related microsoft/winget-pkgs#184815

## Expected behavior

- `Name` property of this package should be `PDF Arranger`.
- `Publisher` property should NOT be `UNKNOWN`.

### PR DESCRIPTION
**Pull request by Dragon1573**
## ✨ What's changed

- Change package name.
- Add package publisher.

## 📝 Additional information

- Resolve #1133
- Resolve microsoft/winget-pkgs#184815

### CODE REVIEW

File: setup_win32.py [hunk 1]
```diff
@@ -175,7 +175,8 @@ def run(self):
         shutil.rmtree(build_exe.build_exe)
 
 
-setup(name='pdfarranger',
+setup(name='PDF Arranger',
+      author='PDF-Arranger developer team',
```
Comment (SpecterShell) [inline setup_win32.py:179]: Maybe you can refer to this file: https://github.com/pdfarranger/pdfarranger/blob/d7d0c52be6076fe3e5964f8b6e4877103d665eec/setup.py#L108

**Comment by kbengs** [comment]
Thanks, maybe you could set author to "The PDF Arranger team" to be consistent with flatpak/flathub, found in:
https://github.com/pdfarranger/pdfarranger/blob/d7d0c52be6076fe3e5964f8b6e4877103d665eec/data/com.github.jeromerobert.pdfarranger.metainfo.xml#L183

It apparently also does change the default folder it is installed to (from "pdfarranger" to "PDF Arranger") I guess that is fine.

File: setup_win32.py (hunk 1 above)
Comment (Dragon1573) [reply setup_win32.py:179]: > [!WARNING]
>
> Rejected by https://github.com/pdfarranger/pdfarranger/pull/1134#issuecomment-2429296836

Comment (Dragon1573) [reply setup_win32.py:179]:
```suggestion
      author='The PDF Arranger team',
```

Suggested by https://github.com/pdfarranger/pdfarranger/pull/1134#issuecomment-2429296836.

**Pull request merged**